package.json

```diff
@@ -88,5 +88,8 @@
     "typedoc-plugin-missing-exports": "^1.0.0",
     "typescript": "^4.7.4"
   },
-  "packageManager": "yarn@3.2.1"
+  "packageManager": "yarn@3.2.1",
+  "dependencies": {
+    "chromedriver": "^120.0.0"
```

Comment:
Please move this dependency to e2e package

Reply:
done here https://github.com/input-output-hk/cardano-js-sdk/pull/1012/commits/a048f61d16b6c6969bdd63ae367f853c9a269e2f

Reply:
It should probably be in `devDependencies` as it's only used for running the tests, and not for anything under `src/`

Reply:
done here https://github.com/input-output-hk/cardano-js-sdk/pull/1012/commits/bacced6cb9d380b38fb7fba77c5489f75518655e